bin/critterget.py

```diff
@@ -70,7 +70,7 @@ def apicall (uri, attribs=None):
 
     try:
         response = requests.get(url, headers=headers)
-        return response.json()
+        return (response.status_code, response.json())
```

Comment:
While I could see this status being useful, it doesn't look like you ever use it.